lua/lspconfig/server_configurations/elmls.lua

```diff
@@ -23,6 +24,7 @@ return {
       disableElmLSDiagnostics = false,
       onlyUpdateDiagnosticsOnSave = false,
     },
+    capabilities = default_capabilities,
```

Comment:
inline ` capabilities = { offsetEncoding =` and also other.

Reply:
Done.